ci/ray_ci/tester.py

```diff
@@ -175,6 +177,7 @@ def main(
     if not bazel_workspace_dir:
         raise Exception("Please use `bazelisk run //ci/ray_ci`")
     os.chdir(bazel_workspace_dir)
+    init_global_config(bazel_runfile("release/ray_release/configs/oss_config.yaml"))
```

Comment:
it is a bit weird that this is referencing from `ray_release`, do we plan to move out this common part or something?

Reply:
yes, let's merge ray_release and ci so they can all share together easily